hooks/subkit

```diff
@@ -39,13 +39,17 @@ if [[ $BOSH_INIT_ENABLED == 1 && $CPI == "warden-cpi" ]]; then
 	exit 1
 fi
 
-for kit in $@; do
-  echo $kit
-  if [[ $BOSH_INIT_ENABLED == 1 && $kit == $CPI ]]; then
-    if [[ -z $CPI_INIT ]]; then
-      echo "bosh-init + $CPI is currently unimplemented" >&2
-      exit 1
+if [[ $BOSH_INIT_ENABLED == 1 ]]; then
+  for kit in $@; do
+    echo $kit
+    if [[ $kit == $CPI ]]; then
+      if [[ -z $CPI_INIT ]]; then
+        echo "bosh-init + $CPI is currently unimplemented" >&2
+        exit 1
+      fi
+      echo $CPI_INIT
     fi
-    echo $CPI_INIT
-  fi
-done
+  done
+else
+  echo recompilable
```

Comment:
Is the intent to always use recompilable (unless its bosh-init)?  any reason we don't want to make it choose-able (I want recompile-able for compatibility vs I want it pre-compiled because im on the compatible stemcell and I want it fast

Reply:
It could go either way - i was just trying to make it easier on existing ops.